File: 백준/Bronze/19572. 가뭄（Small）/가뭄（Small）.js
const [d1, d2, d3] = require('fs').readFileSync(0).toString().trim().split(' ').map(Number);
const total = (d1 + d2 + d3) / 2;

const a = total - d3;
const b = total - d2;
const c = total - d1;

if (a <= 0 || b <= 0 || c <= 0) {
    console.log(-1);
} else {
    console.log(1);
    console.log(`${a.toFixed(1)} ${b.toFixed(1)} ${c.toFixed(1)}`);
}

File: 백준/Bronze/19572. 가뭄（Small）/가뭄（Small）.py
d1, d2, d3 = map(int, input().split())

total = (d1 + d2 + d3) / 2

a = total - d3
b = total - d2
c = total - d1

if a <= 0 or b <= 0 or c <= 0:
    print(-1)
else:
    print(1)
    print(round(a, 1), round(b, 1), round(c, 1))
